hadoop-crypto/src/main/java/com/palantir/hadoop/EncryptedFileSystem.java

```diff
@@ -76,6 +79,29 @@ public FSDataOutputStream create(Path path, FsPermission permission,
         return os;
     }
 
+    @Override
+    public boolean rename(Path src, Path dst) throws IOException {
```

Comment:
is the path guaranteed to be sanitized when it gets here? for instance, what happens if I `rename("foo", "bar//baz")` and then `get("bar/baz")` ?